analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -421,6 +422,16 @@ class Pip(
         return RemoteArtifact(url, Hash.create(hash))
     }
 
+    private fun parseAuthors(pkgInfo: JsonNode): SortedSet<String> =
+        pkgInfo["author"]?.textValue()?.takeIf(::isValidAuthor)?.let { sortedSetOf(it) } ?: sortedSetOf()
+
+    /**
+     * Check if the given [author] string represents a valid author name. There are some non null strings that
+     * indicate that no author information is available. For instance, setup.py files can contain empty strings;
+     * the "pip show" command prints the string "None" in this case.
+     */
+    private fun isValidAuthor(author: String): Boolean = author.isNotBlank() && author.toLowerCase() != "none"
```

Comment:
As `None` is a keyword in Python, I guess it's save to case-sensitively compare to "None".